### PR TITLE
add certificate loading failure error prompt

### PR DESCRIPTION
**Pull request by shzcore**

### CODE REVIEW

File: hubble-be/src/main/java/com/baidu/hugegraph/util/HugeClientUtil.java [hunk 1]
```diff
@@ -74,6 +74,9 @@ public static HugeClient tryConnect(GraphConnection connection) {
             if (message != null && message.startsWith("The version")) {
                 throw new ExternalException("client-server.version.unmatched", e);
             }
+            if (message != null && message.startsWith("Error loading trust store from")) {
+                throw new ExternalException("load.truststore.error", e);
```
Comment (Linary) [inline hubble-be/src/main/java/com/baidu/hugegraph/util/HugeClientUtil.java:78]: load.truststore.error -> https.load.truststore.error

File: hubble-be/src/main/resources/i18n/messages.properties [hunk 2]
```diff
@@ -169,3 +169,4 @@ license.datasize.no-limit=no limit
 license.verify.ip.unauthorized=The hugegraph-hubble's IP {0} doesn't match the authorized {1}
 license.verfiy.mac-unmatch-ip=Failed to get mac address for IP {0}
 license.verify.mac.unauthorized=The hugegraph-hubble's mac {0} doesn't match the authorized {1}
+load.truststore.error=https trusted certificate failed to load
```
Comment (Linary) [inline hubble-be/src/main/resources/i18n/messages.properties:172]: 上下各加一个空行

File: hubble-be/src/main/resources/i18n/messages_zh_CN.properties [hunk 3]
```diff
@@ -169,3 +169,4 @@ license.datasize.no-limit=无限制
 license.verify.ip.unauthorized=hugegraph-hubble 机器的 IP {0} 不在已授权范围内 {1}
 license.verfiy.mac-unmatch-ip=hugegraph-hubble 机器的 MAC 与 IP {0} 不匹配
 license.verify.mac.unauthorized=hugegraph-hubble 机器的 MAC {0} 不在已授权范围内 {1}
+load.truststore.error=https可信证书加载失败
```
Comment (Linary) [inline hubble-be/src/main/resources/i18n/messages_zh_CN.properties:172]: 上面加一个空行

**Review by Linary — CHANGES_REQUESTED** [review]

File: hubble-be/src/main/resources/i18n/messages.properties [hunk 4]
```diff
@@ -169,4 +169,5 @@ license.datasize.no-limit=no limit
 license.verify.ip.unauthorized=The hugegraph-hubble's IP {0} doesn't match the authorized {1}
 license.verfiy.mac-unmatch-ip=Failed to get mac address for IP {0}
 license.verify.mac.unauthorized=The hugegraph-hubble's mac {0} doesn't match the authorized {1}
-load.truststore.error=https trusted certificate failed to load
\ No newline at end of file
+
+https.load.truststore.error=https trusted certificate failed to load
```
Comment (Linary) [inline hubble-be/src/main/resources/i18n/messages.properties:173]: Failed to load https trusted certificate

File: hubble-be/src/main/resources/i18n/messages_zh_CN.properties [hunk 5]
```diff
@@ -169,4 +169,5 @@ license.datasize.no-limit=无限制
 license.verify.ip.unauthorized=hugegraph-hubble 机器的 IP {0} 不在已授权范围内 {1}
 license.verfiy.mac-unmatch-ip=hugegraph-hubble 机器的 MAC 与 IP {0} 不匹配
 license.verify.mac.unauthorized=hugegraph-hubble 机器的 MAC {0} 不在已授权范围内 {1}
-load.truststore.error=https可信证书加载失败
+
+https.load.truststore.error=https可信证书加载失败
```
Comment (Linary) [inline hubble-be/src/main/resources/i18n/messages_zh_CN.properties:173]: https 后加一个空格

**Review by Linary — CHANGES_REQUESTED** [review]

**Review by Linary — APPROVED** [review]

**Review by javeme — APPROVED** [review]

**Pull request merged**